malasakit-django/pcari/models.py

```diff
@@ -268,8 +268,17 @@ class QuantitativeQuestion(Question):
         left_text: The text that is rendered on the left end of the slider.
         right_text: The text that is rendered on the right end of the slider.
     """
+    INPUT_TYPES = (
+        ('range', 'range'),
+        ('number', 'number')
+    )
+
     left_text = models.TextField(blank=True)
     right_text = models.TextField(blank=True)
+    minval = models.PositiveSmallIntegerField(default=0, null=True, blank=True)
+    maxval = models.PositiveSmallIntegerField(default=9, null=True, blank=True)
```

Comment:
One more thing: these fields don't obey the snake case naming convention (i.e. separating words with an underscore, so these should become `min_value` and `max_value`).